src/cascadia/TerminalCore/Terminal.hpp

```diff
@@ -172,12 +172,13 @@ class Microsoft::Terminal::Core::Terminal final :
     SHORT _selectionAnchor_YOffset;
     SHORT _endSelectionPosition_YOffset;
     std::wstring _wordDelimiters;
-    enum selectionExpansionMode
+    enum SelectionExpansionMode

```

Comment:
plz2move this upwards and perhaps out of the class (or at least to somewhere that isn't in the middle of all the members)